src/lib/lmc-maps.ts

```diff
@@ -74,7 +77,20 @@ class LmcMaps {
 
         this.setControls();
 
-        this.hasMarker && createMarker(this.coords).addTo(this.map);
+        this.bounds = new mapboxgl.LngLatBounds();
+
+        this.coords.forEach((coord: mapboxgl.LngLatLike): void => {
+            if (this.hasMarker) {
+                createMarker(coord).addTo(this.map);
+            }
+
+            this.bounds.extend(coord);
```

Comment:
`ERROR in [at-loader] ./src/lib/lmc-maps.ts:87:32 
    TS2345: Argument of type 'LngLatLike' is not assignable to parameter of type 'LngLat | LngLatBounds'.
  Type '{ lng: number; lat: number; }' is not assignable to type 'LngLat | LngLatBounds'.
    Type '{ lng: number; lat: number; }' is missing the following properties from type 'LngLat': wrap, toArray, toBounds`

Reply:
Issue with mapbox-gl: https://github.com/mapbox/mapbox-gl-js/issues/8065

Solved by coordinates conversion, see `mapboxgl.LngLat.convert(coords)`